src/Templatize.coffee

```diff
@@ -47,13 +51,16 @@ descriptionAndArray = (description, array) ->
 		if value instanceof Parameter.__type
 			template.addParameter(value)
 
+		if value instanceof Condition.__type
```

Comment:
```suggestion
		else if value instanceof Condition.__type
```
NBD since they're distinct types.

Reply:
fixed in [c051e58](https://github.com/UnquietCode/Cloud-Temple/pull/2/commits/c051e58ef5d135cb17c4cd6f9f27c1f877a9c6c6)